docs/diagrams/MarkdownTextStylerDiagram.puml

```diff
@@ -0,0 +1,41 @@
+@startuml
+!include style.puml
+
+
+box UI UI_COLOR_T1
+participant ":MainWindow" as MainWindow UI_COLOR
+end box
+box Logic LOGIC_COLOR_T1
+participant ":Command" as Command LOGIC_COLOR
+participant ":CommandResult" as CommandResult LOGIC_COLOR
+end box
+box UI UI_COLOR_T1
+participant ":ResultDisplay" as ResultDisplay UI_COLOR
+participant "<<class>>\nTextStyleHelper" as TextStyleHelper UI_COLOR
+end box
+
+MainWindow -> Command : execute(commandText)
+activate MainWindow
+activate Command
+Command -> CommandResult :execute(commandText)
+activate CommandResult
+CommandResult -->Command :commandResult
+deactivate CommandResult
+Command --> MainWindow : commandResult
+deactivate Command
+
+
+
+MainWindow -> ResultDisplay : resultDisplay(commandResult.text)
+activate ResultDisplay
+
+ResultDisplay -> TextStyleHelper : formattedTextParser(text)
+activate TextStyleHelper
+
+TextStyleHelper --> ResultDisplay : List of styled Text
+deactivate TextStyleHelper
+
+ResultDisplay -> ResultDisplay : Add list of Text to display
+ResultDisplay --> MainWindow
```

Comment:
Does not look like you deactivated the activation bar on ResultDisplay and MainWindow.

Reply:
thanks for the catch, updaetd in https://github.com/AY2122S2-CS2103-W17-1/tp/pull/251/commits/04af621a4f32d0c84a6557320113901da95ef91b